packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -39,6 +39,7 @@ class CupertinoPicker extends StatefulWidget {
     @required this.children,
   }) : assert(diameterRatio != null),
        assert(diameterRatio > 0.0, RenderListWheelViewport.diameterRatioZeroMessage),
+       assert(backgroundColor != null),
```

Comment:
This is allowed to be null. I just checked in some additional documentation to make that clearer.

Reply:
If it is null then the foreground below is null . But foreground is used as a non nullable parameter for a container (IIRC because I'm on my phone)

Reply:
Container(color) can definitely be null (it defaults to null).

We should probably do more clever things when background is null, similar to what I added to handle null background lower down.

Reply:
cc @xster

Reply:
My Bad, I misread the assert about color and decoration.

Reply:
If it makes you feel better, I tried to add this exact same assert in a PR not two days ago. ;-)

Reply:
I removed it from the original PR :D https://github.com/flutter/flutter/pull/14091#pullrequestreview-89212469

Reply:
I removed this new assert.